src/app/components/client/PercentageChart.tsx

```diff
@@ -0,0 +1,132 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+"use client";
+
+import { CSSProperties } from "react";
+import { useL10n } from "../../hooks/l10n";
+import styles from "./Chart.module.scss";
+
+export type WelcomeToPremiumProps = {
+  exposureReduction: number;
+};
+export const PercentageChart = (props: WelcomeToPremiumProps) => {
+  const l10n = useL10n();
+  // const sumOfReduction = props.data.reduce(
+  //   (total, [_label, num]) => total + num,
+  //   0
+  // );
+  // const percentages = props.data.map(([label, num]) => {
+  //   return [label, num / sumOfReduction] as const;
+  // });
```

Comment:
Is this commented code needed for _The Future&trade;_, or should we just ✂️ it out?